src/triage/cli.py

```diff
@@ -200,8 +200,14 @@ def __init__(self, parser):
             action="store_true",
             help="only validate the config file not running Experiment",
         )
+        parser.add_argument(
+            "--profile",
+            action="store_true",
+            dest="profile",
+            help="Record the time spent in various functions using cProfile"
+        )
 
-        parser.set_defaults(validate=True, validate_only=False)
+        parser.set_defaults(validate=True, validate_only=False, profile=False)
```

Comment:
You really don't need to globally set defaults here -- this was added originally to solve a different problem.

If it's important to explicitly set the default `profile=False`, I think it's best to do so where you define the `profile` option; that said, its default is already falsey, so it's not necessary at all.

(While you're at it, the other globally-set defaults, `validate` and `validate_only`, no longer need to be set here either.)